src/database.hpp

```diff
@@ -58,7 +64,12 @@ struct Database
     std::string getstring(const stringid_t stringid) const;
 
     /**
-     * Builds the RTree and reclaims memory by discarding temporary data
+     * Builds the RTree. Needs to be called after
+     * all OSM data parsing has been added.
+     */
+    void build_rtree();
```

Comment:
Can be marked const I think

Reply:
It ends up modifying the `rtree` member of the Database object

Reply:
Ah sorry, you're right 👍